src/translator.h

```diff
@@ -71,6 +71,10 @@ class Translator
       {
         if (latex_command == "latex") latex_command = "pdflatex";
       }
+      if (Config_getBool(LATEX_BATCHMODE))
+      {
+        latex_command +=  " -interaction=batchmode -halt-on-error";
+      }
```

Comment:
Why using `" -interaction=batchmode -halt-on-error";` and not `" -interaction=errorstopmode";`? (I don't know whether or not there is a difference between these 2 implementations).

Implementation should also be in:
- translator_kr.h
- translator_vi.h

Reply:
> 
> 
> Why using `" -interaction=batchmode -halt-on-error";` and not `" -interaction=errorstopmode";`? (I don't know whether or not there is a difference between these 2 implementations).

Important notes from http://tug.org/texinfohtml/latex2e.html
 1.  In errorstop mode (the default), TeX stops at each error and asks for user intervention. In batch mode it **prints nothing on the terminal**, errors are scrolled as if the user hit <return> at every error, and missing files cause the job to abort. 
 2.  -halt-on-error **Stop processing at the first error.** 

Ad 1: Useful, because one sees anything in the refman.log. 
Ad 2: Then also "make" terminates with an error statement. Useful, if make is used in other Makefiles or Bash scripts.

 

> 
> Implementation should also be in:
> 
>     * translator_kr.h
> 
>     * translator_vi.h

Ok!